UnityProject/ProjectSettings/Physics2DSettings.asset

```diff
@@ -3,8 +3,8 @@
 --- !u!19 &1
 Physics2DSettings:
   m_ObjectHideFlags: 0
-  serializedVersion: 4
```

Comment:
why the serialisation version change?

Reply:
That was something automatic on the part of unityengine, the changes to project settings were made through the project settings window. If need be i can make a commit changing back the serializedVersion.